.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -1,47 +1,45 @@
 ---
 name: "\U0001F41E Bug report"
 about: Let us know if something isn't working as you expect it to.
-
+labels: "type: bug" 
 ---
 
-**Describe the bug**
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To reproduce
```

Comment:
Note - I've upgraded all our headings to real heading syntax while here. Just cleaner/simpler and more flexible IMO 😄 (I will admit this is a pet peeve of mine!)